### PR TITLE
Add basic files, including confituration files and package.json

### PR DESCRIPTION
**Pull request by AleksandraCyp**

### CODE REVIEW

File: dist/index.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+/******/ 	"use strict";
+/*!**********************!*\
+  !*** ./src/index.ts ***!
+  \**********************/
+
+document.querySelector('body').style.color = 'red';
+
+/******/ })()
+;
+//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIndlYnBhY2s6Ly9jb2RlcnNjYW1wMjAyMC5wcm9qZWN0LnR5cGVzY3JpcHQucG9rZW1vbmZpZ2h0Ly4vc3JjL2luZGV4LnRzIl0sIm5hbWVzIjpbXSwibWFwcGluZ3MiOiI7Ozs7OztBQUFDLFFBQVEsQ0FBQyxhQUFhLENBQUMsTUFBTSxDQUFxQixDQUFDLEtBQUssQ0FBQyxLQUFLLEdBQUcsS0FBSyIsImZpbGUiOiJpbmRleC5qcyIsInNvdXJjZXNDb250ZW50IjpbIihkb2N1bWVudC5xdWVyeVNlbGVjdG9yKCdib2R5JykgYXMgSFRNTEJvZHlFbGVtZW50KS5zdHlsZS5jb2xvciA9ICdyZWQnIl0sInNvdXJjZVJvb3QiOiIifQ==
```
Comment (lukaszdutka) [inline dist/index.js]: what are those comments? I don't understand :(

File: package.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  "name": "coderscamp2020.project.typescript.pokemonfight",
+  "version": "1.0.0",
+  "description": "**UWAGA! Zaczynając pracę nad projektem — nie róbcie forka.\r Jedna osoba z zespołu (np. Mentor) powinna użyć przycisku `Use this template` i dodać innych członków zespołu jako Collaborators do tego repozytorium.\r Mentorzy mogą dowolnie zmieniać zakres projektu lub zdecydować się na zupełnie inny temat.**",
+  "main": "index.ts",
+  "scripts": {
+    "test": "jest",
+    "test:watch": "jest --watch",
+    "test:cov": "jest --coverage",
+    "build": "webpack",
+    "start": "webpack --watch"
```
Comment (lukaszdutka) [inline package.json:11]: so we decided to use webpack, not parcel? Is there a difference? :P

File: tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,70 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig.json to read more about this file */
+
```
Comment (lukaszdutka) [inline tsconfig.json:4]: I don't know much about those settings, maybe someone from team can check if it's ok?

File: package.json (hunk 2 above)
Comment (lukaszdutka) [reply package.json:11]: It looks like Parcel is more beginner-friendly, with it's autoconfigs:
https://stackoverflow.com/questions/51453462/webpack-vs-parcel

**Review by lukaszdutka — COMMENTED** [review]
Looks ok, but need second approval.

File: tsconfig.json (hunk 3 above)
Comment (daria305) [reply tsconfig.json:4]: Looks good for me, I like the strict rules

File: package.json (hunk 2 above)
Comment (daria305) [reply package.json:11]: It's just that we had webpack in our basic study materials, so it should be already familiar to us

**Review by daria305 — APPROVED** [review]

**Review by daria305 — APPROVED** [review]
The configuration looks good, I don't see any problems

**Review by mariusz-sm — APPROVED** [review]

**Pull request merged**